tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/timeline/timeline_component.css

```diff
@@ -63,8 +63,8 @@ limitations under the License.
   height: 48px;
   padding: 0;
   position: absolute;
-  right: 0;
-  width: 440px;
+  right: 40px;
+  width: calc(100% - 420px);
```

Comment:
Write a comment of where this magical `420` is coming from. Also, I personally think `left` is more readable in this case.

Reply:
Done. Using `left` now.